src/main/java/org/folio/ebsconet/controller/TenantController.java

```diff
@@ -4,7 +4,7 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-@RestController("folioTenantController")
 @RequestMapping(value = "/_/")
+@RestController("folioTenantController")
```

Comment:
In OrdersController for example we have @RestController at first , and @RequestMapping after. Better to have the same style across controllers. No changes needed here, just thing about the feasibility of such changes